Assets/AppCenter/Editor/AppCenterPreBuild.cs

```diff
@@ -49,6 +55,48 @@ public void OnPreprocessBuild(BuildTarget target, string path)
         }
     }
 
+#if UNITY_ANDROID
+    public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid settingsMaker)
+    {
+        var loaderZipFile = AppCenterSettingsContext.AppCenterPath + "/Plugins/Android/appcenter-loader-release.aar";
+        var loaderFolder = "appcenter-loader-release";
+        var settingsFilePath = "appcenter-loader-release/res/values/appcenter-settings.xml";
```

Comment:
folder name could be appended from loaderFolder, and maybe use constants for all of these paths